{"asctime": "2020-03-26 17:29:01,445", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class FieldingAreaMarker"}
{"asctime": "2020-03-26 17:29:03,567", "levelname": "DEBUG", "message": "ApplyDataModel :FieldingAreaMarker created"}
{"asctime": "2020-03-26 17:29:03,569", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingAreaMarker"}
{"asctime": "2020-03-26 17:29:03,570", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-26 17:29:05,341", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingAreaMarker"}
{"asctime": "2020-03-26 17:29:05,343", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comments"}
{"asctime": "2020-03-26 17:29:06,934", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comments added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingAreaMarker"}
{"asctime": "2020-03-26 17:29:06,935", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:29:07,344", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:29:07,344", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:29:07,345", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:29:07,347", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'FieldingPointMarker', 'geometry': 'POINT', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'fieldingpointmarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-26 17:29:07,539", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class FieldingPointMarker"}
{"asctime": "2020-03-26 17:29:09,399", "levelname": "DEBUG", "message": "ApplyDataModel :FieldingPointMarker created"}
{"asctime": "2020-03-26 17:29:09,400", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingPointMarker"}
{"asctime": "2020-03-26 17:29:09,402", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-26 17:29:10,930", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingPointMarker"}
{"asctime": "2020-03-26 17:29:10,930", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comments"}
{"asctime": "2020-03-26 17:29:12,718", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comments added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingPointMarker"}
{"asctime": "2020-03-26 17:29:12,719", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:29:13,195", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:29:13,196", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:29:13,197", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:29:13,199", "levelname": "DEBUG", "message": "ApplyDataModel :{'name': 'FieldingLineMarker', 'geometry': 'POLYLINE', 'spatialref': 'AlbersEqualAreaConic', 'fields': 'fieldinglinemarker.json', 'globalids': True, 'editortracking': True, 'archiving': True, 'attachments': True}"}
{"asctime": "2020-03-26 17:29:13,476", "levelname": "DEBUG", "message": "ApplyDataModel :Creating feature class FieldingLineMarker"}
{"asctime": "2020-03-26 17:29:16,050", "levelname": "DEBUG", "message": "ApplyDataModel :FieldingLineMarker created"}
{"asctime": "2020-03-26 17:29:16,051", "levelname": "DEBUG", "message": "ApplyDataModel :Applying fields for C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingLineMarker"}
{"asctime": "2020-03-26 17:29:16,052", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Label"}
{"asctime": "2020-03-26 17:29:18,238", "levelname": "DEBUG", "message": "ApplyDataModel :Field Label added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingLineMarker"}
{"asctime": "2020-03-26 17:29:18,238", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Comments"}
{"asctime": "2020-03-26 17:29:20,290", "levelname": "DEBUG", "message": "ApplyDataModel :Field Comments added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingLineMarker"}
{"asctime": "2020-03-26 17:29:20,290", "levelname": "DEBUG", "message": "ApplyDataModel :Applying OBJECTID"}
{"asctime": "2020-03-26 17:29:20,727", "levelname": "DEBUG", "message": "ApplyDataModel :Adding field OBJECTID failed"}
{"asctime": "2020-03-26 17:29:20,728", "levelname": "ERROR", "message": "ApplyDataModel :Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:29:20,728", "levelname": "ERROR", "message": "Traceback as follows:", "exc_info": "Traceback (most recent call last):\n  File \"c:\\Cloud\\Dropbox\\Work\\Axin\\Python\\CreateFeatureClass\\ApplyDataModel.py\", line 160, in _process_fields\n    arcpy.AddField_management(fc,fd['name'],fd['type'],fd['precision'],fd['scale'],fd['length'],fd['alias'],fd['nullable'],fd['required'],fd['domain'])\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4170, in AddField\n    raise e\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\management.py\", line 4167, in AddField\n    retval = convertArcObjectToPythonObject(gp.AddField_management(*gp_fixargs((in_table, field_name, field_type, field_precision, field_scale, field_length, field_alias, field_is_nullable, field_is_required, field_domain), True)))\n  File \"C:\\Program Files\\ArcGIS\\Pro\\Resources\\ArcPy\\arcpy\\geoprocessing\\_base.py\", line 511, in <lambda>\n    return lambda *args: val(*gp_fixargs(args, True))\narcgisscripting.ExecuteError: Failed to execute. Parameters are not valid.\nWARNING 000012: OBJECTID already exists\nERROR 000800: The value is not a member of Text | Float (single precision) | Double (double precision) | Short (small integer) | Long (large integer) | Date | Blob (binary data) | Raster imagery | GUID (globally unique identifier).\nFailed to execute (AddField).\n"}
{"asctime": "2020-03-26 17:29:20,730", "levelname": "DEBUG", "message": "ApplyDataModel :Applying Type"}
{"asctime": "2020-03-26 17:29:22,699", "levelname": "DEBUG", "message": "ApplyDataModel :Field Type added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingLineMarker"}
{"asctime": "2020-03-26 17:29:22,700", "levelname": "DEBUG", "message": "ApplyDataModel :Applying LineType"}
{"asctime": "2020-03-26 17:29:24,529", "levelname": "DEBUG", "message": "ApplyDataModel :Field LineType added to C:/Cloud/Dropbox/Work/Axin/rdt.fielding.datamodel.gdb/FieldingLineMarker"}
{"asctime": "2020-03-26 17:29:24,530", "levelname": "DEBUG", "message": "ApplyDataModel :Processing feature classes finished."}
